lamindb/core/storage/objects.py

```diff
@@ -1,62 +1,95 @@
 from __future__ import annotations
 
 from pathlib import PurePosixPath
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, TypeAlias, TypeVar
 
 from anndata import AnnData
 from pandas import DataFrame
 
 if TYPE_CHECKING:
     from lamindb_setup.core.types import UPathStr
 
+SpatialData = TypeVar("SpatialData")
+MuData = TypeVar("MuData")
 
-def _mudata_is_installed():
-    try:
-        import mudata  # noqa: F401c
-    except ImportError:
-        return False
-    return True
+SupportedDataTypes: TypeAlias = AnnData | DataFrame | MuData | SpatialData
```

Comment:
Nice! But let's please call these `DataObjectTypes` or `OTypes` or `ObjectTypes` or `SupportedObjectTypes` to match the convention `Artifact.otype`, which holds the string serialization of the class name of the object.

Reply:
Yah, I was actually thinking of adding a `ScverseDataStructures` type to a `types.py` module at the root level of lamindb. This would help us type a bit better across a lot more functions.

Reply:
Sounds great!